uiServices/src/testSuiteServices_test.go

```diff
@@ -480,3 +480,63 @@ func TestGetAllSuitesNoHeader(t *testing.T) {
 
 	assert.Equal(t, http.StatusBadRequest, w.Code, "Did not get all test suites")
 }
+
+func TestSuccessfulDelete(t *testing.T) {
+	r := chi.NewRouter()
+	r.Mount("/", GetIndexPage())
+
+	filePath := os.Getenv("GOPATH") + "/src/github.com/xtracdev/automated-perf-test/uiServices/test/"
+	request, err := http.NewRequest(http.MethodDelete, "/test-suites/TestSuiteService", nil)
+
+	request.Header.Set("testSuitePathDir", filePath)
+	request.Header.Get("testSuitePathDir")
+
+	w := httptest.NewRecorder()
+	r.ServeHTTP(w, request)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	assert.Equal(t,http.StatusNoContent, w.Code, "Error. Did not successfully Delete")
+}
+
+func TestDeleteFileNotFound(t *testing.T) {
```

Comment:
What about permission problems. Can you test that too.

Reply:
Our application makes the files. I don't know how it would happen that we wouldn't have permission to delete them. How would I replicate this situation for a test where I wouldn't have permission?

Reply:
you could do this in sys tests I think? Have a function that creates a file in the step with no read permissions, and then use the service to try to delete it

Reply:
How do you create a file with no read permissions?